### ISSUE TITLE
Feature Request: Log bundle download

### ISSUE THREAD

**Issue by agracey** [epinio/ui#94]
### Is your feature request related to a problem?

To allow for easier supportability, we need a way to get all of the right logs in a concise way. 

### Solution you'd like

A button in the UI that pulls together all the relevant logs:

- Epinio server (and previous container)
- Epinio ui (and previous container)
- Recent staging Jobs
- Container registry
- Minio

And optionally, a way to add all the application logs. 

### Alternatives you've considered

We could just document all the places to pull logs, but a single click would obviously be significantly easier for the end user!

**Comment by jimmykarily** [comment]
By "all application logs", you mean the logs of every application in a single button click? On a cluster with many applications this may be a lot. Also, this will have to work on the Kubernetes level, Epinio isn't that "self aware" currently. For example, it generally doesn't know and shouldn't care if there is Minio or other S3 compatible storage running. To collect the logs from Minio, it would have to know it's there.

All the logs listed above, are the output of some Pod. The person interested for those logs is the operator if I understand this correctly. Isn't it sane for the operator to setup some more generic logging solution (like fluentd) to forward the logs to some log collecting framework that allows filtering and such? Isn't this how operators watch logs anyway?

If there is something we can do in Epinio to make this easier, we should do it. But we don't want to re-invent the wheel here.

**Comment by juadk** [comment]
It sounds like the support bundle which is used for Harvester / Rancher.
https://docs.harvesterhci.io/v1.0/troubleshooting/harvester/
https://github.com/rancher/support-bundle-kit

**Comment by agracey** [comment]
Yes, exactly like the support bundles. I didn't know where those were documented to link. Thank you!

And as far as the application logs. Those should not be included by default (That's likely a huge increase in size and number of requests) but it might be nice to have a way to pull them as well?
